src/sentry/static/sentry/app/views/settings/organization/general/organizationGeneralSettingsView.jsx

```diff
@@ -51,7 +51,7 @@ const OrganizationGeneralSettingsView = createReactClass({
           data.access.indexOf('org:admin') === -1 &&
```

Comment:
unrelated but, can't we be using [`includes`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/includes)?

Reply:
Yeah we could